addons/website_slides/controllers/mail.py

```diff
@@ -56,9 +31,15 @@ def mail_update_message(self, thread_model, thread_id, message_id, post_data, **
         self._portal_post_check_attachments(attachment_ids, post.get('attachment_tokens', []))
 
         pid = int(post['pid']) if post.get('pid') else False
-        if not _check_special_access(thread_model, thread_id, post.get('token'), post.get('hash'), pid):
+        channel = request.env[thread_model]._get_thread_with_access(
```

Comment:
`thread_model` is forced to `slide.channel` above so we might as well hard-code it to `"slide.channel"` to make the code more readable.